dvc/utils/serialize/_py.py

```diff
@@ -0,0 +1,96 @@
+import ast
+
+from funcy import reraise
+
+from ._common import ParseError, _load_data
+
+
+class PYFileCorruptedError(ParseError):
+    def __init__(self, path, message="PY file structure is corrupted"):
+        super().__init__(path, message)
+
+
+def load_py(path, tree=None):
+    return _load_data(path, parser=parse_py, tree=tree)
+
+
+def parse_py(text, path):
+    """Parses text from .py file into Python structure."""
+    with reraise(SyntaxError, PYFileCorruptedError(path)):
+        tree = ast.parse(text, filename=path)
+
+    result = ast_tree_to_dict(tree)
+    return result
+
+
+def ast_tree_to_dict(tree):
```

Comment:
Let's make this an internal API as we do `*` imports.
https://github.com/iterative/dvc/blob/79e8e4edafb8a4745200f0a62f49c29ef074bdea/dvc/utils/serialize/__init__.py#L6

```suggestion
def _ast_tree_to_dict(tree):
```

Reply:
Applies to all other functions except `load_py` and ` parse_py`.